src/cpp/session/modules/NotebookData.R

```diff
@@ -92,9 +92,16 @@
       options,
       file = output)
 
-    .Call("rs_recordData", output, list(classes = className,
-                                        nrow = nRow, 
-                                        ncol = nCol))
+    # standard metadata
+    metadata <- list(classes = className, nrow = nRow, ncol = nCol, summary = list())
+
+    # if tibble is loaded, use it to create a summary of the object
+    if (isNamespaceLoaded("tibble"))
```

Comment:
```suggestion
    if ("tibble" %in% loadedNamespaces())
```

`isNamespaceLoaded()` was added in R 3.2.0, and so is a bit too new for us to use.